komodo/reverse_dep_graph.py

```diff
@@ -58,13 +58,13 @@ def _dump_dot(reverse, pkg, version, out):
 def _dump_dot_dep(reverse, pkg, version, out, seen):
     if pkg in seen:
         return
-    id = pkg.lower().replace("-", "_")
-    out.write(f'  {id} [label="{pkg}-{version}"];\n')
+    _id = pkg.lower().replace("-", "_")
```

Comment:
I think it's more conventional to use a _trailing_ underscore to avoid a collision with a builtin. That's what PEP8 says at least. (The leading underscore signals 'internal use' --- which this is too, I guess.)

Reply:
fixed!